src/gnatss/ops/qc.py

```diff
@@ -69,16 +52,77 @@ def _compute_ticks_and_labels(
     """
     time_min = data[time_col].min()
     time_max = data[time_col].max()
-    step = (time_max - time_min) / n_ticks
-    ticks = np.arange(time_min, time_max + step, step)
-    labels = np.apply_along_axis(_sec_to_iso, 0, ticks)
+    time_delta = time_max - time_min
+
+    # step defines the interval in seconds between x-axis ticks
+    step = None
+
+    # Standard intervals of 1s, 5s, 30s, 1min, 5min, 30min, 1hr, 5hr
```

Comment:
Looks like this is different from that issue #185 requests:

"This issue propose a change to have the tick steps by hourly if the difference b/w min and max is more than 1 hour, or be minutes if the difference is less than an hour with n_ticks no more than the specified n_ticks."
"No more that 15 number of ticks!"

ie, I interpret it as something like this:
```python
MAX_INTERVALS = 15
if time_delta <= SECS_IN_HOUR:
    num_intervals = max(MAX_INTERVALS, time_delta/SECS_IN_MINUTE)
    fmt =  "%Y-%m-%dT%H:%M"
else:
    num_intervals = max(MAX_INTERVALS, time_delta/SECS_IN_HOUR)
    fmt =  "%Y-%m-%dT%H:00"

step = ceil(time_delta/num_intervals)
```

Reply:
Hello @carlosgjs . The main reason I did not add ticks as you mentioned above is I wanted to add tick labels at "good to read" intervals steps like 1s, 30s, 1min, 1hr etc.

Reply:
Thanks for the great interpretation @carlosgjs. Looks like this is now implemented, but with even finer resolution of seconds, when finer precision is required.